### DISCUSSION TITLE
Is this a open source? Can I make these in Canada?

### DISCUSSION THREAD

**Discussion by Biker1bob** [willyvmm/mouSTer#41] [General]
Just wondered if there was a possibility to make and resell these in Canada North America. 
I see there is no stock in the Retrohack store. What is the status of this? I see no source for making boards? So I guess this is a private project?

**Comment by willyvmm** [comment]
Hi.
This is not an open-source project.
I'm working on in-sourcing the production to be more flexible.

**Reply by Biker1bob** [reply]
Ok, I am interested in a bulk buy maybe to resell here in Canada/ North America. Shipping times, cost and exchange makes ordering just one not viable.

Let me know.

**Reply by willyvmm** [reply]
Hi.
The new production will be available around the beginning of July. You are very welcome to become a reseller in Canada/North America. 
In that case, the MOQ is 100 pcs.
Drop me an email to my nick here (at) google email so we can continue the discussion offline :)

Best Regards
willy.
